tests/unit_tests/plugins/auth-pam/Makefile.am

```diff
@@ -0,0 +1,11 @@
+AUTOMAKE_OPTIONS = foreign
+
+check_PROGRAMS = testdriver
```

Comment:
This should probably be wrapped in `if ENABLE_PLUGIN_AUTH_PAM` (like in src/plugins/auth-pam/Makefile.am), so that the tests only run in we are actually compiling the plugin.  (Why not always run plugin tests? Well, perhaps libpam is not available, but we do want to run `make check`.)